APIClient/Services/IServices.cs

```diff
@@ -32,8 +32,9 @@ public interface IServices
         /// <param name="filePath">Path and name of the attachment file.</param>
         /// <param name="asset">Asset to save the attachment to.</param>
         /// <param name="attachmentName">The name of the attachment.</param>
+        /// <param name="useAbsolutePathAsFileName">Option to use the Ansolute Path as the FileName. Defaults to true.</param>
         /// <returns>Oid</returns>
-        Oid SaveAttachment(string filePath, Asset asset, string attachmentName);
+        Oid SaveAttachment(string filePath, Asset asset, string attachmentName, bool useAbsolutePathAsFileName = true);
```

Comment:
Changing an existing signature is an interface-breaking change (even with default parameters, which don't do what you'd like them to do).

Instead, I suggest adding a new overload with the flag.